termwiz/src/lineedit/mod.rs

```diff
@@ -163,6 +163,12 @@ impl<'term> LineEditor<'term> {
         }
     }
 
+    pub fn with_text(terminal: &'term mut dyn Terminal, default: &str) -> Self {
+        let mut editor = Self::new(terminal);
+        editor.line.insert_text(default);
+        return editor;
+    }
+
```

Comment:
I prefer to avoid changing the public interface for this, as there is already a method that can be used to update the contents of the editor

```suggestion
```